build/yamls/base/antctl.yml

```diff
@@ -47,3 +56,9 @@ subjects:
     # ServiceAccount token.
     name: antrea-controller
     namespace: kube-system
+  - kind: ServiceAccount
+    # This is to allow antctl to be executed inside the Antrea agent Pod
```

Comment:
Sometimes you used "agent" and sometimes "Controller". To be consistent, maybe capitalize the first character always.

Reply:
Done.